index.js

```diff
@@ -1,27 +1,159 @@
 const Recipe = require('./models/Recipe.model');
-// Import of the data from './data.json'
 const data = require('./data');
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
 
-// Connection to the database "recipe-app"
-mongoose
-  .connect(MONGODB_URI, {
-    useCreateIndex: true,
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
-  .then(self => {
-    console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+
+
+// 2. MIDDLEWERS
+
+app.use(express.static("public"))
+
+app.set("views", __dirname + "/views");
+app.set("view engine", "hbs");
+
+hbs.registerPartials(__dirname + "/views/partials")
+
+connectDB()
+
+
+
+// 3. RUTAS
+
+// CRUD: Patron de operaciones, que nos permite manipular datos de un proyecto de software
+// (Crear, leer, actualizar, borrar)
+
+
+
+app.get("/home", (req, res) => {
+  res.render("home")
+})
+
+
+
+
+app.get("/c-recipe", async (req, res) => {
+
+  const myNewRecipe = {
+
+    title: "Taco random",
+    level: "Promaster300",
+    ingridients: ["Mole", "Tortilla", "Sal", "Carne", "Pimienta", "Chile Morron"],
+    cuisine: "Mexasian",
+    dishType: "Platillo Principal",
+    image: "https://i.ytimg.com/vi/LTffYgdWs8M/maxresdefault.jpg",
+    duration: 200,
+    creator: "Chef Javier"
+  }
+
+
+  // ITERACION.2
+  // Crea mi receta en la base de datos
+
+  const recipeCreated = await Recipe.create(myNewRecipe)
+
+  res.render("c-recipe", {
+
+    data: recipeCreated
+ 
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+})
+
+
+// ITERACION.3
+// Crea multiples recetas en la base de datos
+
+ 
+app.get("/c-recipes", async (req, res) => {
+
+  const recipesCreated = await Recipe.insertMany(data)
+  console.log(recipesCreated)
+
+  res.render("c-recipes")
+
+})  
+
+
+
+
+//  ITERACION.4
+// Find the user with id '61973da6701c132a1892eee7' and update its duration -=120 = 100 total;
+
+Recipe.findById('61973da6701c132a1892eee7')
+  .then(Recipe => {
+    Recipe.duration = 100;
+    return Recipe.save(); // Update the user '61973da6701c132a1892eee7' and return a promise
   })
-  .catch(error => {
-    console.error('Error connecting to the database', error);
-  });
+  .then(Recipe => console.log('The user was updated: ' + Recipe))
+  .catch(err => console.log('An error occurred:', err));
+
+
+  // MUESTRA ESTO EN TERMINAL PERO NO EN MONDODB
+  /* The user was updated: {
+    ingridients: [],
+    _id: 61973da6701c132a1892eee7,
+    title: 'Rigatoni alla Genovese',
+    level: 'Easy Peasy',
+    cuisine: 'Italian',
+    dishType: 'main_course',
+    image: 'https://images.media-allrecipes.com/userphotos/720x405/3489951.jpg',
+    duration: '100', <------------------- ACTUALIZADO
+    creator: 'Chef Luigi',
+    __v: 0
+} */
+
+
+
+
+
+
+//  ITERACION.5
+
+Recipe.findByIdAndRemove('619744154791d5303454b00b')
```

Comment:
ibas super bien solo que el mismo caso el id no depende de nosotros mongo los genera, tambien tienes concatenedo dos metodos para eliminar te sugiero utilizar 1, me parece mejor en este caso deleteOne por que no necesita el id puede ser una propiedad como lo tenias 😉 
```suggestion
Recipe.deleteOne({ title: "Carrot Cake" })
  .then((Recipe) => console.log("The user was deleted: " + Recipe.title))
  .catch((err) => console.log("An error occurred:", err));
```